README.md

```diff
@@ -86,6 +86,7 @@ These projects have been developed mainly in Toronto, or by Torontoians
 - [Inclusive Design Institute](https://github.com/inclusive-design) - R&D for inclusiveness and diversity. [Website](https://inclusivedesign.ca/)
 - [The Heap](https://github.com/the-heap/) - Cultivated + managed community projects especially welcome to new contributors.
 - [Toronto Mesh](https://github.com/tomeshnet) - Community mesh network in Toronto. [Website](https://tomesh.net)
+- [yorkulibraries](https://github.com/yorkulibraries) [Website](www.library.yorku.ca)
```

Comment:
I would say change `www.library.yorku.ca` to `https://www.library.yorku.ca/web/` - as there is a 301 redirect - to prevent awesome_bot complains in the future.

Reply:
I grabbed it from the GitHub profile page. I would just allow 301s on awesome-bot, it's not a blocking error.

Reply:
Great. I would say proceed with this PR and if in following days the issue with https://bentomiso.com/ wasn't resolved we can remove it.

Reply:
Actually if you don't remove the problematic URL GH pages won't get updated! So you might want to remove it in this PR and ping the owner in an issue.

Reply:
Because it's being updated in Travis after awesome_bot finishes, and if it fails the whole pipeline fails.

Reply:
Compare the last two commits in `master` branch for example.

Reply:
I'd rather our build didn't depend upon other websites that way. It seems awfully brittle.